docs/DeveloperGuide.md

```diff
@@ -60,6 +60,26 @@ The figure below represents the sequence diagram when the user wants to add a st
 
 <>
 
+### Model Component
+The `model` package contains the `InstrumentManager` class and the main parent class `Instrument`. It is defined
```

Comment:
Avoid saying main parent class<- its not exactly the main, instrumentManager class and instrument are both like equally important and do different stuff. Can just say contains both the InstrumentManager and Instrument class